internal/host/hostcommands/download_boot_artifacts_cmd.go

```diff
@@ -59,6 +59,7 @@ func (c *downloadBootArtifactsCmd) GetSteps(ctx context.Context, host *models.Ho
 		return nil, fmt.Errorf("failed to generate urls for DownloadBootArtifactsRequest: %w", err)
 	}
 	// Reclaiming a host is only used in the operator scenario (not SaaS) so other auth types don't need to be considered
+	// might need to add agent-install-local
```

Comment:
Same here, this is only for when we're rebooting a host back into the infraenv after it was installed so you should never have to care about that use case.